UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/HealthCheck/AzureWebJobsHealthCheckClient.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using UKHO.ExchangeSetService.Common.Configuration;
+using UKHO.ExchangeSetService.Common.Helpers;
+using UKHO.ExchangeSetService.Common.Models.Enums;
+
+namespace UKHO.ExchangeSetService.Common.HealthCheck
+{
+    [ExcludeFromCodeCoverage]
```

Comment:
Is there a reason why we cant test this class/methods?

Reply:
Fixed, As calling external client excluded this class have made the feasible changes to cover unit test now.